utils/gyb_sourcekit_support/UIDs.py

```diff
@@ -182,6 +182,7 @@ def __init__(self, internal_name, external_name):
     KEY('CompletionCheckDependencyInterval',
         'key.completion_check_dependency_interval'),
     KEY('AnnotatedTypename', 'key.annotated.typename'),
+    KEY('CompileOperation', 'key.compile_operation'),
```

Comment:
It doesn't need to be part of this PR, but could you document the new key in tools/SourceKit/docs/Protocol.md?